src/vs/workbench/contrib/scm/browser/repositoryPane.ts

```diff
@@ -796,6 +785,12 @@ export class RepositoryPane extends ViewPane {
 		this._register(this.inputEditor.onDidFocusEditorText(() => addClass(editorContainer, 'synthetic-focus')));
 		this._register(this.inputEditor.onDidBlurEditorText(() => removeClass(editorContainer, 'synthetic-focus')));
 
+		const updateInputEditorOptions = () => {
+			const newEditorOptions = this.getInputEditorOptions();
+			this.inputEditor.updateOptions(newEditorOptions);
+		};
+		Event.filter(this.configurationService.onDidChangeConfiguration, e => e.affectsConfiguration('git.commitMessageFontFamily'))(updateInputEditorOptions);
```

Comment:
Love it. But we should collect the listener disposable so we cleanup once the repository gets disposed.